molgenis-app/src/main/java/org/molgenis/app/WebAppDatabasePopulatorServiceImpl.java

```diff
@@ -62,8 +62,11 @@ public void populateDatabase()
 		// Genomebrowser stuff
 		Map<String, String> runtimePropertyMap = new HashMap<String, String>();
 
+		//test
+		runtimePropertyMap.put("resourcePath","alternative");
```

Comment:
Define a static final String somewhere for resourcePath to prevent code breaking on name change.